WIP-CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 
 - Upgraded falco-exporter chart version to `v0.9.6` and app version to `v0.8.3`
 - Added "IPs in Subnet mask" logic in checkIfDiffAndUpdateDNSIPs, checkIfDiffAndUpdateKubectlIPs and checkIfDiffAndUpdateIPs of update-ips.bash
+- Upgrade kube-prometheus-stack chart version from `v45.2.0` to `v49.2.0` and app version from `v0.63.0` to `v0.67.1`
```

Comment:
any user facing changes that we and they should be aware?

Reply:
None as far as i'm aware !!